### ISSUE TITLE
[XF]Can not absolute navigation in UWP

### ISSUE THREAD

**Issue by nuitsjp** [PrismLibrary/Prism#932]
### Package info
- Platform: UWP、Windows、WinPhone
- Prism version: 6.2.0、6.3.0-pre1
- Xamarin version (if applicable): 4.2.2.11
- Windows 10 SDK version (if applicable): 10.0; Build 14393

### Repro steps
If NavigateAsync specifies string beginning with a slash.

```cs
_navigationService.NavigateAsync("/NavigationPage/ContentPage");
```
At this time, absolute navigation in Android and iOS.
But in UWP（and Windows and WinPhone） it is relative navigation.

I think that this is not an appropriate behavior.
If so, the cause is here.
https://github.com/PrismLibrary/Prism/blob/master/Source/Xamarin/Prism.Forms/Navigation/PageNavigationService.cs#L92

```cs
System.Uri uri = new Uri("/NavigationPage/ContentPage", UriKind.RelativeOrAbsolute)"
```
At this time, "uri.IsAbsoluteUri" is true on Android and iOS.
But, "uri.IsAbsoluteUri" is false on UWP.
This is as designed as a platform.

Therefore, need to generate the Uri by looking at the path like Like UriParsingHelper#EnsureAbsolute.
https://github.com/PrismLibrary/Prism/blob/master/Source/Xamarin/Prism.Forms/Common/UriParsingHelper.cs#L75

I fixed the source on this matter.
I send pull request.
Would appreciate feedback.
Thank you.

**Comment by brianlagunas** [comment]
Great find!  You just need a small fix to your logic and I will merge the PR.

**Comment by brianlagunas** [comment]
Fixed with your PR #933
